### ISSUE TITLE
check_yum.py on Redhat8

### ISSUE THREAD

**Issue by HermansenOle** [HariSekhon/Nagios-Plugins#366]
./check_yum.py -vvv
Nagios Plugin for Yum updates on RedHat/CentOS systems - Version 0.11.3
Author: Hari Sekhon

setting plugin timeout to 30 seconds
running command: /usr/bin/yum --security check-update
Returncode: '100'
Output: 'Updating Subscription Management repositories.
Last metadata expiration check: 2:20:37 ago on Thu 08 Apr 2021 07:32:41 AM CEST.

NetworkManager.x86_64                     1:1.26.0-14.el8_3            sb-baseos
NetworkManager-config-server.noarch       1:1.26.0-14.el8_3            sb-baseos
NetworkManager-libnm.x86_64               1:1.26.0-14.el8_3            sb-baseos
NetworkManager-team.x86_64                1:1.26.0-14.el8_3            sb-baseos
NetworkManager-tui.x86_64                 1:1.26.0-14.el8_3            sb-baseos
kernel.x86_64                             4.18.0-240.22.1.el8_3        sb-baseos
kernel-core.x86_64                        4.18.0-240.22.1.el8_3        sb-baseos
kernel-modules.x86_64                     4.18.0-240.22.1.el8_3        sb-baseos
kernel-tools.x86_64                       4.18.0-240.22.1.el8_3        sb-baseos
kernel-tools-libs.x86_64                  4.18.0-240.22.1.el8_3        sb-baseos
systemd.x86_64                            239-41.el8_3.2               sb-baseos
systemd-libs.x86_64                       239-41.el8_3.2               sb-baseos
systemd-pam.x86_64                        239-41.el8_3.2               sb-baseos
systemd-udev.x86_64                       239-41.el8_3.2               sb-baseos
'
running command: yum updateinfo
Returncode: '0'
Output: 'Updating Subscription Management repositories.
Last metadata expiration check: 2:20:40 ago on Thu 08 Apr 2021 07:32:41 AM CEST.
Updates Information Summary: available
    1 Security notice(s)
        1 Important Security notice(s)
    6 Bugfix notice(s)
    1 Enhancement notice(s)
'
output lines not accounted for: ['NetworkManager.x86_64                     1:1.26.0-14.el8_3            sb-baseos', 'NetworkManager-config-server.noarch       1:1.26.0-14.el8_3            sb-baseos', 'NetworkManager-libnm.x86_64               1:1.26.0-14.el8_3            sb-baseos', 'NetworkManager-team.x86_64                1:1.26.0-14.el8_3            sb-baseos', 'NetworkManager-tui.x86_64                 1:1.26.0-14.el8_3            sb-baseos', 'kernel.x86_64                             4.18.0-240.22.1.el8_3        sb-baseos', 'kernel-core.x86_64                        4.18.0-240.22.1.el8_3        sb-baseos', 'kernel-modules.x86_64                     4.18.0-240.22.1.el8_3        sb-baseos', 'kernel-tools.x86_64                       4.18.0-240.22.1.el8_3        sb-baseos', 'kernel-tools-libs.x86_64                  4.18.0-240.22.1.el8_3        sb-baseos', 'systemd.x86_64                            239-41.el8_3.2               sb-baseos', 'systemd-libs.x86_64                       239-41.el8_3.2               sb-baseos', 'systemd-pam.x86_64                        239-41.el8_3.2               sb-baseos', 'systemd-udev.x86_64                       239-41.el8_3.2               sb-baseos']
YUM WARNING: Yum output signature (14 unique lines) is larger than number of total updates (6). Please make sure you have upgraded to the latest version from https://github.com/harisekhon/nagios-plugins. If the problem persists, please raise a ticket at https://github.com/harisekhon/nagios-plugins/issues with the full -vvv output

**Comment by jmlarabauche** [comment]
Same issue in CentOS 8.

**Comment by martinbalint** [comment]
Same issue in Centos 7
```
Nagios Plugin for Yum updates on RedHat/CentOS systems - Version 0.11.3
Author: Hari Sekhon

setting plugin timeout to 30 seconds
running command: /usr/bin/yum --security check-update
Returncode: '100'
Output: 'Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: d36uatko69830t.cloudfront.net
 * epel: d2lzkl7pfhq30w.cloudfront.net
 * extras: d36uatko69830t.cloudfront.net
 * updates: d36uatko69830t.cloudfront.net
...
 --> pcp-export-pcp2spark-4.3.2-13.el7_9.x86_64 from updates excluded (updateinfo)
 --> pcp-export-pcp2xml-4.3.2-13.el7_9.x86_64 from updates excluded (updateinfo)
 --> pcp-export-pcp2influxdb-4.3.2-13.el7_9.x86_64 from updates excluded (updateinfo)
 --> pcp-export-pcp2json-4.3.2-13.el7_9.x86_64 from updates excluded (updateinfo)
 --> pcp-export-pcp2zabbix-4.3.2-13.el7_9.x86_64 from updates excluded (updateinfo)
 --> pcp-export-zabbix-agent-4.3.2-13.el7_9.x86_64 from updates excluded (updateinfo)
6 package(s) needed for security, out of 6 available

kernel.x86_64                         3.10.0-1160.24.1.el7               updates
kernel-tools.x86_64                   3.10.0-1160.24.1.el7               updates
kernel-tools-libs.x86_64              3.10.0-1160.24.1.el7               updates
libldb.x86_64                         1.5.4-2.el7                        updates
pyldb.x86_64                          1.5.4-2.el7                        updates
python-perf.x86_64                    3.10.0-1160.24.1.el7               updates
'
output lines not accounted for: ['6 package(s) needed for security, out of 6 available', 'kernel.x86_64                         3.10.0-1160.24.1.el7               updates', 'kernel-tools.x86_64                   3.10.0-1160.24.1.el7               updates', 'kernel-tools-libs.x86_64              3.10.0-1160.24.1.el7               updates', 'libldb.x86_64                         1.5.4-2.el7                        updates', 'pyldb.x86_64                          1.5.4-2.el7                        updates', 'python-perf.x86_64                    3.10.0-1160.24.1.el7               updates']
YUM WARNING: Yum output signature (7 unique lines) is larger than number of total updates (6). Please make sure you have upgraded to the latest version from https://github.com/harisekhon/nagios-plugins. If the problem persists, please raise a ticket at https://github.com/harisekhon/nagios-plugins/issues with the full -vvv output

```

**Comment by rascalDan** [comment]
Same again, CentOS 7, same output as @martinbalint, different packages.

**Comment by tjyang** [comment]
- check_yum.py 0.11.3 on Fedora 34 is ok.
<details>
[me@ipa01 nagios-plugins]$ ./check_yum.py -D -t 120
Nagios Plugin for Yum updates on RedHat/CentOS systems - Version 0.11.3
Author: Hari Sekhon

setting plugin timeout to 120 seconds
running command: /usr/bin/yum --security check-update
Returncode: '0'
Output: 'Last metadata expiration check: 0:15:29 ago on Sat 11 Sep 2021 04:00:19 PM CDT.
'
YUM OK: 0 Security Updates Available. 0 Non-Security Updates Available | security_updates_available=0 non_security_updates_available=0 total_updates_available=0
[me@ipa01 nagios-plugins]$ cat /etc/redhat-release
Fedora release 34 (Thirty Four)
[me@ipa01 nagios-plugins]$

</details>

**Comment by pbalu-code** [comment]
I have made a fix for this issue, when the number of updates is not matching with the number of lines.
Pull request: #379
It may have to test. It is working on Oracle linux.

**Comment by tjyang** [comment]
I confirm this one-line fix on RockLinux 8.5. Thank you @pbalu-code.

**Comment by HariSekhon** [comment]
I've just pushed a fix for DNF format changes, please git pull and try the latest version 12.2.

@pbalu-code @tjyang Careful, that PR change has a regex bug that will blind you to all the security packages. I've updated it to work properly instead.

I've tested this on Rocky Linux 8, Alma Linux 8 and CentOS 8.
